Makefile

```diff
@@ -0,0 +1,15 @@
+PREFIX ?= /usr/local
+
+all:
+	cargo build --release
+
+install:
+	cargo build --release
+	install -d ${PREFIX}/bin
+	install -c -m 555 target/release/snare ${PREFIX}/bin/snare
+	install -d ${PREFIX}/man/man1
```

Comment:
These manual paths are only correct for OpenBSD. In Linux you'd install into `${PREFIX}/share/man`.

Is it worth fixing? Maybe not.

Reply:
I don't see a sensible way of making this fully generic without going down the autoconf path. To be honest, anyone who notices that will probably be capable of editing the `Makefile` paths.

Reply:
I good and easy compromise is to add a `MAN_PREFIX` variable like you did for `PREFIX`.

Reply:
Or you can check the OS in the makefile without autoconf. Admittedly, that's nicer/easier for gmake.

Reply:
OK, I think https://github.com/softdevteam/snare/pull/35/commits/92592c5604598f43aa3e8c13a5348fe21cfb3241 might be the trick?

Reply:
That'll do the trick!